libs/coin-modules/coin-stacks/src/errors.ts

```diff
@@ -1,3 +1,3 @@
-import { createCustomErrorClass } from "@ledgerhq/errors";
+import { createCustomErrorClass } from "@ledgerhq/errors/";
```

Comment:
why `/` at the end of `"@ledgerhq/errors/"; ?

Reply:
done in ddf4dc51bae9251a780f176cfbd9f197142d9ef0